.test_durations

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
Guess this needs a gitignore

Reply:
Nope, it's required, but it could probably live under tests/